modules/ossm-federation-config-import.adoc

```diff
@@ -0,0 +1,134 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-config-import_{context}"]
+= Importing a service into a federated mesh
+
+Importing services lets you explicitly specify which services exported from other meshes that you want to allow into your service mesh.
+
+//Insert ImportedServiceSet diagram here
+
+You use an `ImportedServiceSet` resource to select services for import. Only services exported by a mesh peer and explicitly imported are available to the mesh. Services that you do not explicitly import are not made available within the mesh.
+
+* You can select services by namespace or name.
+* You can use wildcards to select services, for example, to import all the services that were exported to the namespace.
+* You can select services for export using a label selector, which may be global to the mesh, or scoped to a specific member namespace.
+* You can import services using an alias. For example, you can import the `custom-ns/bar` service as `other-mesh/bar`.
+// Need non foo/bar example above
+* You can specify a custom domain suffix, which will be appended to the `name.namespace` of an imported service for its fully qualified domain name; for example, `other-mesh/bar.imported.local`.
+
+The following example is for the `green-mesh` importing a service that was exported by `red-mesh`.
+
+.Example ImportServiceSet
+[source,yaml]
+----
+kind: ImportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: red-mesh #name of mesh that exported the service
+  namespace: green-mesh-system #mesh namespace that service is being imported into
+spec:
+  importRules: # first matching rule is used
+  # import ratings.bookinfo as ratings.bookinfo
+  - type: NameSelector
+    nameSelector:
+      importAsLocal: false
+      namespace: bookinfo
+      name: ratings
+      alias:
+        # services will be imported as <name>.mesh2-bookinfo.svc.mesh1-imports.local
+        namespace: bookinfo
+        name: ratings
+----
+
+.ImportServiceSet parameters
+[options="header"]
+[cols="l, a, a"]
+|===
+|Parameter |Description |Values
+|metadata:
+  name:
+|Name of the ServiceMeshPeer that exported the service to the federated mesh.
+|
+
+|metadata:
+  namespace:
+|Name of the ServiceMeshPeer namespace where the service is being imported.
+|
+
+|spec:
+  importRules:
+    -type:
+|Type of rule that will govern the import for the service. The first matching rule found for the service will be used for the import.
+|`NameSelector`, `LabelSelector`
```

Comment:
Only `NameSelector` is supported.  There is no `LabelSelector` for `ImportedServiceSet`.